### ISSUE TITLE
Progress bar broken

### ISSUE THREAD

**Issue by Korijn** [conda/conda#3492]
![image](https://cloud.githubusercontent.com/assets/1882046/18741247/576de78c-80ae-11e6-8604-2af7117b9cdd.png)

```
C:\Users\Korijn\dev\myproject>conda info
Current conda install:

               platform : win-64
          conda version : 4.2.7
       conda is private : False
      conda-env version : 4.2.7
    conda-build version : 2.0.1
         python version : 3.5.1.final.0
       requests version : 2.9.1
       root environment : C:\Users\Korijn\Miniconda3  (writable)
    default environment : C:\Users\Korijn\Miniconda3
       envs directories : C:\Users\Korijn\Miniconda3\envs
          package cache : C:\Users\Korijn\Miniconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                          https://repo.continuum.io/pkgs/free/noarch/
                          https://repo.continuum.io/pkgs/pro/win-64/
                          https://repo.continuum.io/pkgs/pro/noarch/
                          https://repo.continuum.io/pkgs/msys2/win-64/
                          https://repo.continuum.io/pkgs/msys2/noarch/
            config file : C:\Users\Korijn\.condarc
           offline mode : False
```

**Comment by alanhdu** [comment]
I am also having this problem, although only with `conda env` commands (e.g. `conda env create` or `conda env update`). `conda install` works fine.

```bash
$ conda info
Current conda install:

               platform : linux-64
          conda version : 4.2.7
       conda is private : False
      conda-env version : 4.2.7
    conda-build version : 2.0.1
         python version : 3.5.2.final.0
       requests version : 2.11.1
       root environment : /home/alan/anaconda  (writable)
    default environment : /home/alan/anaconda/envs/test
       envs directories : /home/alan/anaconda/envs
          package cache : /home/alan/anaconda/pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64/
                          https://conda.anaconda.org/conda-forge/noarch/
                          https://conda.anaconda.org/conda-canary/linux-64/
                          https://conda.anaconda.org/conda-canary/noarch/
                          https://repo.continuum.io/pkgs/free/linux-64/
                          https://repo.continuum.io/pkgs/free/noarch/
                          https://repo.continuum.io/pkgs/pro/linux-64/
                          https://repo.continuum.io/pkgs/pro/noarch/
            config file : /home/alan/.condarc
           offline mode : False
```

**Comment by kalefranz** [comment]
resolved with #3524
